docs/writing-rules/rule-syntax.md

```diff
@@ -759,6 +760,78 @@ int("2147483646")
 
 This removes quotes (`'`, `"`, and `` ` ``) from both ends of the metavariable content. As a result, Semgrep detects `"2147483648"`, but it does **not** detect `"2147483646"`. This is useful when you expect strings to contain integer or float data.
 
+### `metavariable-name`
+
+:::info
+`metavariable-name` requires a Semgrep account and **[cross-file analysis](/docs/getting-started/cli#enable-cross-file-analysis)**. 
```

Comment:
Seems redundant with the updated version at lines 771-3? Cross-file isn't required though, so if we want to keep this it should either just say Semgrep account or Pro engine.

Reply:
You do not _need_ cross-file. But, the results are better with cross-file enabled.

The pro engine is a requirement. This is not quite the same as being logged-in, since you can pass, e.g., `--oss-only`.